tests/unit/expire.tcl

```diff
@@ -882,17 +880,12 @@ start_cluster 1 0 {tags {"expire external:skip cluster"}} {
             fail "bgsave did not stop in time."
         }
 
-        # Verify dict is under rehashing
-        set htstats [r debug HTSTATS 0]
-        assert_match {*rehashing target*} $htstats
-
         # put some data into slot 12182 and trigger the resize
         r psetex "{foo}0" 500 a
 
         # Verify dict rehashing has completed
-        set htstats [r debug HTSTATS 0]
-        wait_for_condition 20 100 {
-            ![string match {*rehashing target*} $htstats]
+        wait_for_condition 10 100 {
+            [string match {*table size: 4*} [r debug HTSTATS 0]]
```

Comment:
there could be multiple tables, maybe make sure we match the specific one by adding more details to the pattern? like
`[Dictionary HT]*table size: 4*number of elements: 4*[Expires HT]`?
or actually, in this case it's unnecessary since none of the other dicts are expected to be completely empty?

Reply:
Not sure if I exactly followed you. Note: in cluster-enabled setup the HTSTATS is aggregated for all the dictionaries and response is sent as 1. 
Both main HT/expire HT are at size 4 and no. of keys are at 2 at the end before the expiration. I could make it more explicit but I don't think it has any significance.